Tools-Override/FrameworkTargeting.targets

```diff
@@ -228,11 +228,18 @@
 
   <Target Name="BinPlaceFiles"
           Condition="'@(BinPlaceDir)' != ''"
-          DependsOnTargets="GetBinPlaceItems" >
+          DependsOnTargets="GetBinPlaceItems"
+          Inputs="@(BinPlaceDir);%(BinPlaceDir.ItemName)"
```

Comment:
Actually this is incorrect.  If you use metadata in a condition on a target you get the following error:
```
References to item metadata are not allowed in target conditions unless they are part of an item transform.
```

The correct way to trigger target batching is to use inputs as i have done.